docker/run.sh

```diff
@@ -238,6 +238,16 @@ function runserver {
     su cchq -c "./manage.py runserver $@ 0.0.0.0:8000"
 }
 
+function runcelery {
+    JS_SETUP=yes setup python
+    su cchq -c "celery -A corehq worker -l info"
```

Comment:
This will run `celery` queue only. Is the intention here just to run one queue?

And if we want to account for periodic tasks, then we would also have to run `django-celery-beat` service. https://docs.celeryq.dev/en/latest/userguide/periodic-tasks.html#using-custom-scheduler-classes

Reply:
That's right. These are intentionally omitted. 

A bit of background on this is that there is no real requirement from within our developer team to do these from inside Docker. Our recommended [way](https://github.com/dimagi/commcare-hq/blob/master/DEV_SETUP.md#running-celery) is to run celery and pillowtop is from the host machine itself.

These are being added to satisfy a SolTech external deliverable where all the required services needed to be run within Docker itself.

If we require these in the future, we will add support for that too.

Reply:
Understood. Thanks for sharing that info.